### DISCUSSION TITLE
main error

### DISCUSSION THREAD

**Discussion by yawnicorn** [sst-inc/help#40] [Q&A - iOS] [answered]
<img width="866" alt="Screenshot 2021-10-15 at 1 15 32 PM" src="https://user-images.githubusercontent.com/92565561/137435907-c7c7d75d-5956-4bfe-9a49-da031d6045e5.png">
it happens when I try to click a button to perform a segue. How do I fix it?

**Comment by granwyntan** [comment, marked as answer]
Hi, @yawnicorn
You could try checking if there are existing connections with a warning symbol ⚠️ in your connections inspector, and reconnect the IBAction to your button. If the issue still persists, could you zip up your xcode project and files and send it here. Thanks

**Reply by yawnicorn** [reply]
Ive tried reconnecting the IBAction and there are no warning symbols for that but the error is still there. Here is the zipped up project
[Clicker App v2.zip](https://github.com/sst-inc/help/files/7351092/Clicker.App.v2.zip)
.

**Reply by yawnicorn** [reply]
Ok nvm I fixed it
